### DISCUSSION TITLE
Use only part of the suggestion

### DISCUSSION THREAD

**Discussion by yuri-reigo** [community/community#121655] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Sometimes the suggestion of Co-pilot is good, but only until a certain point. Would be great to have a way to use only a part of the suggestion, maybe by using a mouse to indicated desired end of snippet to complete.

**Comment by davevad93** [comment]
Hi @yuri-reigo ,
There is a way in copilot to use only a part of the suggestion, in VS Code when a code suggestion appears press the Ctrl Key and then the Right Arrow Key on the keyboard, this way you can accept the code suggestion "word by word":

![capture](https://github.com/community/community/assets/94308234/744629ac-74b9-4662-bc4a-7fc5a7deb4bb)

Maybe you already knew this and it's not exactly what you're looking for, but I'll tell you anyway just in case.
